### DISCUSSION TITLE
Yubikey

### DISCUSSION THREAD

**Discussion by Lazybone** [Natizyskunk/vscode-sftp#423] [Q&A] [answered]
is it possible to specify a Yubikey that has an SSH key in the config file?

**Comment by Natizyskunk** [comment, marked as answer]
@Lazybone Could you open an issue respecting [this](https://github.com/Natizyskunk/vscode-sftp/issues/new?assignees=&labels=&projects=&template=feature_request.md&title=) template please ?
